content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,306 @@
+consolidate Nodes. This means that they need to be explicitly integrated with each supported cloud
+provider. The performance and feature set of a given autoscaler can differ between cloud provider
+integrations.
+
+{{< mermaid >}}
+graph TD
+    na[Node autoscaler]
+    k8s[Kubernetes]
+    cp[Cloud Provider]
+
+    k8s --> |get Pods/Nodes|na
+    na --> |drain Nodes|k8s
+    na --> |create/remove resources backing Nodes|cp
+    cp --> |get resources backing Nodes|na
+
+    classDef white_on_blue fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+    classDef blue_on_white fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+    class na blue_on_white;
+    class k8s,cp white_on_blue;
+{{</ mermaid >}}
+
+### Autoscaler implementations
+
+Cluster autoscaler and Karpenter are the two Node autoscalers currently sponsored by
+[SIG Autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling).
+
+From the perspective of a cluster user, both autoscalers should provide a similar Node autoscaling
+experience. Both will provision new Nodes for unschedulable Pods, and both will consolidate the
+Nodes that are no longer optimally utilized.
+
+Different autoscalers may also provide features outside the Node autoscaling scope described on this
+page, and those additional features may differ between them.
+
+Consult the sections below, and the linked documentation for the individual autoscalers to decide
+which autoscaler fits your use-case better.
+
+#### Cluster Autoscaler
+
+Cluster Autoscaler adds or removes Nodes to pre-configured _Node groups_. Node groups generally map
+to some sort of cloud provider resource group (most commonly a Virtual Machine group). A single
+instance of Cluster Autoscaler can simultaneously manage multiple Node groups. When provisioning,
+Cluster Autoscaler will add Nodes to the group that best fits the requests of pending Pods. When
+consolidating, Cluster Autoscaler always selects specific Nodes to remove as opposed to just
+resizing the underlying cloud provider resource group.
+
+Additional context:
+
+* Documentation overview:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md
+* Supported cloud providers:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#faqdocumentation
+* Cluster Autoscaler FAQ:
+  https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md
+* Contact: https://github.com/kubernetes/community/tree/master/sig-autoscaling#contact
+
+#### Karpenter
+
+Karpenter auto-provisions Nodes based on [NodePool](https://karpenter.sh/docs/concepts/nodepools/)
+configuration provided by the cluster operator. Karpenter handles all aspects of node lifecycle, not
+just autoscaling. It works directly with individual cloud provider resources (most commonly
+individual Virtual Machines), and doesn't rely on cloud provider resource groups.
```

Comment:
I see you call out "most commonly individual VMs" a lot. I wonder if there's reason to call this out because I'm imagining that it's assumed.